scripts/delayed.js

```diff
@@ -715,3 +715,11 @@ if (otId) {
     loadBlock(adsBlock);
   }
 }
+
+// load the delayed iframes, mainly used by the promotion block
+const iFrames = document.querySelectorAll('iframe[data-src]');
```

Comment:
ultimately, I'm not sure if this is better or worse than the intersection observer approach, could argue pros/cons of each. PSI score seems roughly the same either way, though tee-times page maybe does slightly better with this approach?


In short, think this could use a second opinion.